packages/story-editor/src/app/currentUser/currentUserProvider.tsx

```diff
@@ -17,25 +17,33 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
 import { useCallback, useEffect, useState } from '@googleforcreators/react';
-
+import type { ReactNode } from 'react';
 /**
  * Internal dependencies
  */
 import { useAPI } from '../api';
+import type { User } from '../../types/configProvider';
+import type {
+  CurrentUserState,
+  updateCurrentUserProps,
+} from '../../types/currentUserProvider';
 import Context from './context';
 
-function CurrentUserProvider({ children }) {
-  const [currentUser, setCurrentUser] = useState({});
+export interface CurrentUserProviderProps {
+  children: ReactNode;
+}
+
+function CurrentUserProvider({ children }: CurrentUserProviderProps) {
```

Comment:
In other places (`APIProvider`, `HistoryProvider`) we started using `PropsWithChildren` instead to get this type for free. Let's use that here for consistency as well.

```ts
import type { PropsWithChildren } from 'react';

function CurrentUserProvider({
  children,
}: PropsWithChildren<Record<string, never>>) {

// ...
```

And yes in other parts of the code base we should probably start doing that as well, but that's for another day.